src/App.js

```diff
@@ -5,13 +5,13 @@ import AddBookForm from './components/form';
 
 function App() {
   return (
-    <>
+    <section className="container">
       <Nav />
       <Routes>
         <Route path="/form" element={<AddBookForm />} />
         <Route path="/Categories" element={<Categories />} />
       </Routes>
-    </>
+    </section>
```

Comment:
- [ ] when first opening the application the the page is blank, only the navbar is visible as you can see below, this is due to the fact that the root path is not setup. 
![afbeelding](https://user-images.githubusercontent.com/39240820/232134163-2ffe6d5f-2f1c-472a-8855-50e0dc081d34.png)


You can fix this by doing the following:
```
      <Routes>
        <Route path="/" element={<AddBookForm />} />
        <Route path="/form" element={<AddBookForm />} />
        <Route path="/Categories" element={<Categories />} />
      </Routes>
```